api/specs.proto

```diff
@@ -106,8 +106,11 @@ message TaskSpec {
 		ContainerSpec container = 1;
 	}
 
+	// ContainerHostConfigSpec specifies hostconfig for container.
+	ContainerHostConfigSpec hostconfig = 2;
+
 	// Resource requirements for the container.
-	ResourceRequirements resources = 2;
```

Comment:
Field numbers can no longer be changed without breaking backwards compatibility.